app-header-layout/app-header-layout.html

```diff
@@ -181,7 +181,6 @@
         var rect = this.getBoundingClientRect();
         this._getFixedNodes().forEach(function(node) {
           node.style.left = this.hasScrollingRegion ? '' : rect.left + 'px';
-          node.style.width = this.hasScrollingRegion ? '' : rect.width + 'px';
```

Comment:
This will not work when `<app-drawer>` is set to `align="end"`.  In the `right drawer` case, we need to set `right` to the drawer's width value.